src/mono/mono/mini/mini-wasm.c

```diff
@@ -722,24 +722,24 @@ ssize_t sendfile(int out_fd, int in_fd, off_t *offset, size_t count);
 
 ssize_t sendfile(int out_fd, int in_fd, off_t *offset, size_t count)
```

Comment:
Why is it done here and not at a higher level with better exception type and less code?

Reply:
These are libc stubs they should error out instead of asserting.

Reply:
Yeah I agree.